apps/easypid/src/locales/pt/messages.json

```
@@ -0,0 +1,3906 @@
+	},
+	"didcommCancelFlow.verification.description": {
+		"message": "If you stop, no data will be shared.",
+		"placeholders": {},
+		"comments": ["Confirmation description when cancelling a proof request"],
+		"origin": [["src/features/didcomm/utils.ts", 21]],
+		"translation": "Se tu parar, nenhum dado será compartilhado."
+	},
+	"slideWizard.confirmation.stopSharingDescription": {
+		"message": "If you stop, no data will be shared.",
+		"placeholders": {},
+		"comments": [
+			"Description in confirmation dialog about stopping data sharing"
+		],
+		"origin": [["../../packages/app/src/components/SlideWizard.tsx", 55]],
+		"translation": "Se tu parar, nenhum dado será compartilhado."
+	},
+	"didcommCancelFlow.connect.description": {
+		"message": "If you stop, nothing will be saved.",
+		"placeholders": {},
+		"comments": [
+			"Confirmation description when cancelling a connection request"
+		],
+		"origin": [["src/features/didcomm/utils.ts", 31]],
+		"translation": "Se tu parar, nada será salvo."
+	},
+	"receiveCredential.stopDescription": {
+		"message": "If you stop, the card offer will be cancelled.",
+		"placeholders": {},
+		"comments": [],
+		"origin": [
+			["src/features/receive/FunkeOpenIdCredentialNotificationScreen.tsx", 542]
+		],
+		"translation": "Se tu parar, a oferta do cartão será cancelada."
+	},
+	"pidSetup.stopDescription": {
+		"message": "If you stop, you can still do the setup later.",
+		"placeholders": {},
+		"comments": [],
+		"origin": [["src/features/pid/FunkePidSetupScreen.tsx", 477]],
+		"translation": "Se tu parar, ainda poderá fazer a configuração mais tarde."
+	},
+	"inbox.header": {
+		"message": "Inbox",
+		"placeholders": {},
+		"comments": ["Header title for the inbox screen"],
+		"origin": [["src/features/didcomm/InboxScreen.tsx", 35]],
+		"translation": "Caixa de entrada"
+	},
+	"submission.offlineRequestDescription": {
+		"message": "Information about the verifier could not be shown. Carefully consider if you trust this organization.",
+		"placeholders": {},
+		"comments": [
+			"Message shown when the request is offline and verifier is unknown"
+		],
+		"origin": [["src/features/share/slides/ShareCredentialsSlide.tsx", 95]],
+		"translation": "As informações sobre o verificador não puderam ser mostradas. Considere cuidadosamente se tu confia nesta organização."
+	},
+	"common.presentationShared": {
+		"message": "Information has been successfully shared.",
+		"placeholders": {},
+		"comments": [
+			"Toast or message shown when a presentation was successfully shared"
+		],
+		"origin": [["../../packages/translations/src/commonMessages.ts", 109]],
+		"translation": "As informações foram compartilhadas com sucesso."
+	},
+	"common.informationRequestDeclined": {
+		"message": "Information request has been declined.",
+		"placeholders": {},
+		"comments": [],
+		"origin": [["../../packages/translations/src/commonMessages.ts", 137]],
+		"translation": "A solicitação de informações foi recusada."
```

```suggestion
		"translation": "O pedido de informações foi recusada."
```

Use "pedido" instead of "solicitação"